src/reference-data/defaults/fare-product-type-config.json

```diff
@@ -173,5 +173,101 @@
       "offerEndpoint": "zones",
       "requiresLogin": true
     }
+  },
```

Comment:
@oyvindmikkelsenatb Hey :), I'm wondering what do you think about those texts for boat tickets and the translations.

Reply:
@ckbilstad Could you have a look at what @jorelosorio is mentioning since you've made the designs?

Reply:
@Sebstorvik Bruker vi bare “boat” om hurtigbåt på engelsk?

Reply:
@jorelosorio Some adjustments:
«Når du skal reise én gang» = «When you travel occasionally»

«Når du reiser litt oftere» = «When you travel more frequently»

Reply:
Bare "boat" ja. Sjekket på atb.no og hørte med Vegard (marked) som mente det samme. Boat & ferry er de vi bruker henholdsvis om hurtigbåt og ferge

Reply:
Fixed